bastionai/examples/distilbert_example_notebook.ipynb

```diff
@@ -45,7 +45,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install bastionai"
+    "!pip install bastionlab"
```

Comment:
Don't run examples from bastionai, there are not maintained.
Checking whether the notebooks in docs work is enough for now